kubernetes/spack/internal-spack-io/gh-gl-sync/config-maps.yaml

```diff
@@ -0,0 +1,218 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gh-gl-sync
+  namespace: spack
+data:
+  entrypoint.sh: |
+    sh /scripts/install-packages.sh
+    export GITLAB_SSH_KEY_BASE64="$( base64 < /secrets/gitlab-ssh-key )"
+    export GITHUB_TOKEN="$( cat /secrets/github-access-token )"
+    # TODO: update these repos after testing
+    bash /scripts/github-prs-to-gitlab.sh \
+      'scottwittenburg/gitlab-ci-tests' \
+      'ssh.gitlab.spack.io' \
+      'scott/gitlab-ci-tests'
+
+  install-packages.sh: |
+    apk add bash git openssh python3
+
+  github-list-prs.py: |
+    #!/usr/bin/env python3
+
+    import sys
+    import argparse
+    import urllib.request
+    import json
+
+    parser = argparse.ArgumentParser(description="List open PRs")
+    parser.add_argument("-t", "--token", help="GitHub access token")
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument("-o", "--open", help="List open PRs",
+                       action="store_true", default=False)
+    group.add_argument("-c", "--closed", help="List closed PRs",
+                       action="store_true", default=False)
+    group.add_argument("-a", "--all", help="List all PRs",
+                       action="store_true", default=False)
+    parser.add_argument("repo", help="GitHub repo (org/repo or user/repo)")
+    args = parser.parse_args()
+
+    if args.open:
+        state = "open"
+    elif args.closed:
+        state = "closed"
+    elif args.all:
+        state = "all"
+    else:
+        state = "open"
+
+    try:
+        request = urllib.request.Request(
+            'https://api.github.com/repos/%s/pulls?state=%s' % (args.repo, state))
+        if args.token:
+            request.add_header('Authorization', 'token %s' % args.token)
+        response = urllib.request.urlopen(request)
+    except OSError:
+        sys.exit(1)
+
+    prs = json.loads(response.read())
+
+    for pr in prs:
+        print("%d %s" % (pr['number'], pr['head']['ref']))
+
+  github-prs-to-gitlab.sh: |
+    #!/usr/bin/env bash
```

Comment:
I don't actually know if this is bash specific or not.  It may be generic `sh` compliant, I haven't tried it though.